diff --git a/src/apps/common/submodules/buyNumbers/buyNumbers.js b/src/apps/common/submodules/buyNumbers/buyNumbers.js
index 6a8bcd08..aa6b5852 100644
--- a/src/apps/common/submodules/buyNumbers/buyNumbers.js
+++ b/src/apps/common/submodules/buyNumbers/buyNumbers.js
@@ -65,12 +65,12 @@ define(function(require){
 					'US': {
 						local: true,
 						toll_free: [
-							'80*',
-							'84*',
-							'85*',
-							'86*',
-							'87*',
-							'88*'
+							'800',
+							'844',
+							'855',
+							'866',
+							'877',
+							'888'
 						],
 						vanity: true,
 						prefix: 1,
